### PR TITLE
journalist: flake8 compliance

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

makes journalist.py flake8 compliant

## Testing

* pytest tests

* pip install git+https://github.com/berkerpeksag/astor
* checkout develop and run cd securedrop ; python -m astor.rtrip . ; mv tmp_rtrip tmp_rtrip.orig
* checkout this pull request and run cd securedrop ; python -m astor.rtrip .
* diff -ru tmp_rtrip tmp_rtrip.orig
<pre>
diff -ru tmp_rtrip.orig/journalist.py tmp_rtrip/journalist.py
--- tmp_rtrip.orig/journalist.py	2017-07-07 09:43:01.081228354 +0200
+++ tmp_rtrip/journalist.py	2017-07-07 09:43:29.213265795 +0200
@@ -1,17 +1,17 @@
-import os
 from datetime import datetime
 import functools
 from flask import Flask, request, render_template, send_file, redirect, flash, url_for, g, abort, session
 from flask_wtf.csrf import CSRFProtect
 from flask_assets import Environment
-from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
+from sqlalchemy.orm.exc import NoResultFound
 from sqlalchemy.exc import IntegrityError
+from sqlalchemy.sql.expression import false
 import config
 import version
 import crypto_util
 import store
 import template_filters
-from db import db_session, Source, Journalist, Submission, Reply, SourceStar, get_one_or_else, WrongPasswordException, LoginThrottledException, InvalidPasswordLength
+from db import db_session, Source, Journalist, Submission, Reply, SourceStar, get_one_or_else, LoginThrottledException, InvalidPasswordLength
 import worker
 app = Flask(__name__, template_folder=config.JOURNALIST_TEMPLATES_DIR)
 app.config.from_object(config.JournalistInterfaceFlaskConfig)
@@ -449,7 +449,7 @@
     for sid in cols_selected:
         id = Source.query.filter(Source.filesystem_id == sid).one().id
         submissions += Submission.query.filter(Submission.downloaded ==
-            False, Submission.source_id == id).all()
+            false(), Submission.source_id == id).all()
     if submissions == []:
         flash('No unread submissions in collections selected!', 'error')
         return redirect(url_for('index'))
@@ -580,7 +580,7 @@
 def download_unread_sid(sid):
     id = Source.query.filter(Source.filesystem_id == sid).one().id
     submissions = Submission.query.filter(Submission.source_id == id, 
-        Submission.downloaded == False).all()
+        Submission.downloaded == false()).all()
     if submissions == []:
         flash('No unread submissions for this source!')
         return redirect(url_for('col', sid=sid))
</pre>

Comparing the sources normalized with the AST ensures white space changes do not modify the source behavior

### CODE REVIEW

**Comment by dachary** [comment]
The travis error seems unrelated. Is it a known transient error ?

<pre>
=================================== FAILURES ===================================

______________ TestJournalistApp.test_download_unread_all_sources ______________

self = <tests.test_journalist.TestJournalistApp testMethod=test_download_unread_all_sources>

    def test_download_unread_all_sources(self):

        self._bulk_download_setup()

        self._login_user()

    

        # Download all unread messages from all sources

        self.resp = self.client.post(

            '/col/process',

            data=dict(action='download-unread',

                      cols_selected=[self.source0.filesystem_id,

                                     self.source1.filesystem_id]))

    

        # The download request was succesful, and the app returned a zipfile

>       self.assertEqual(self.resp.status_code, 200)

E       AssertionError: 302 != 200

tests/test_journalist.py:772: AssertionError
</pre>

**Comment by dachary** [comment]
that was an actual bug, with sqlalchemy False is not really false() ;-) Fixed  	daa2e22 and repushed

File: .travis.yml [hunk 1]
```diff
@@ -58,6 +58,6 @@ script:
   # to alter the pip requirements, which would cause config tests to fail.
   - pip install -r securedrop/requirements/develop-requirements.txt
   - make docs-lint
-  - pushd securedrop; flake8 db.py crypto_util.py manage.py store.py secure_tempfile.py source.py template_filters.py tests/test_2fa.py tests/conftest.py tests/test_db.py tests/test_manage.py tests/test_crypto_util.py tests/test_journalist.py tests/test_integration.py && popd
+  - pushd securedrop; flake8 db.py crypto_util.py journalist.py manage.py store.py secure_tempfile.py source.py template_filters.py tests/test_2fa.py tests/conftest.py tests/test_db.py tests/test_manage.py tests/test_crypto_util.py tests/test_journalist.py tests/test_integration.py; popd
```
Comment (redshiftzero) [inline .travis.yml]: I think we want this `&& popd` there (in the context of #1931)

Comment (conorsch) [inline .travis.yml]: You don't need to switch working directories to execute flake8, it'll handle full path names. So you can just loop over the whitelisted files, or add a Makefile target (e.g. #1920).

Comment (dachary) [inline .travis.yml]: yeah, this is a temporary thing until we can just flake8 with no argument when all files are good

**Comment by dachary** [comment]
@redshiftzero nice catch ! fixed and repushed :-)

**Comment by redshiftzero** [comment]
Ah, can you resolve conflicts and repush?

**Comment by dachary** [comment]
ooops, forgot to resolve, sorry about that. rebased & repushed

**Comment by redshiftzero** [comment]
Hm, it looks like the Travis builds are failing here due to #1971. Rebasing once more should fix (this should be the last rebase :pray:)

**Comment by dachary** [comment]
:-) rebased & repushed

**Pull request merged**